config/auth_u2m.go

```diff
@@ -0,0 +1,67 @@
+package config
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"os/exec"
+	"strings"
+
+	"github.com/databricks/databricks-sdk-go/logger"
+	"golang.org/x/oauth2"
+)
+
+type BricksCliCredentials struct {
+}
+
+func (c BricksCliCredentials) Name() string {
+	return "bricks-cli"
+}
+
+func (c BricksCliCredentials) Configure(ctx context.Context, cfg *Config) (func(*http.Request) error, error) {
+	if !cfg.IsAws() {
+		return nil, nil
+	}
+	ts := bricksCliTokenSource{cfg}
+	_, err := ts.Token()
+	if err != nil {
+		if strings.Contains(err.Error(), "databricks OAuth is not") {
+			// OAuth is not configured or not supported
+			return nil, nil
+		}
+		if strings.Contains(err.Error(), "executable file not found") {
+			logger.Debugf("Most likely Bricks CLI is not installed")
+			return nil, nil
+		}
+		return nil, err
+	}
+	logger.Infof("Using Bricks CLI authentication with Databricks OAuth tokens")
+	return refreshableVisitor(&ts), nil
+}
+
+type bricksCliTokenSource struct {
+	cfg *Config
+}
+
+func (ts *bricksCliTokenSource) Token() (*oauth2.Token, error) {
+	what := ts.cfg.Host
+	if ts.cfg.IsAccountClient() {
+		what = fmt.Sprintf("%s/oidc/accounts/%s", ts.cfg.Host, ts.cfg.AccountID)
+	}
+	out, err := exec.Command("bricks", "auth", "token", what).Output()
```

Comment:
We should have a config option and environment variable for the location of the bricks CLI. 
Actually we should do this for any CLI.

Reply:
let's start with this one. which variable do you use for bricks CLI?